libnmstate/schema.py

```diff
@@ -222,3 +224,82 @@ class OVSBridge(object):
 class OVSBridgePortType(object):
     SYSTEM = 'system'
     INTERNAL = 'internal'
+
+
+class Team(object):
+    TYPE = InterfaceType.TEAM
+    CONFIG_SUBTREE = 'team'
+
+    SLAVES = 'slaves'
+    CONFIG = 'config'
```

Comment:
Since we don't have actual use case on Team yet, maybe we just start with the minimum `slaves` as the `json config`
is too complex do design without use case.

Will `slaves` only works?

Reply:
Yes it works, done!